### ISSUE TITLE
53510806 - Final Fantasy XIII-2

### ISSUE THREAD

**Issue by krikic** [xenia-project/game-compatibility#947]
[Marketplace](http://marketplace.xbox.com/es-ES/Product/FINAL-FANTASY-XIII-2/66acd000-77fe-1000-9115-d80253510806)

Tested on (xenia commit link)

# Issues:
Crash after load the game
# Log:
[xenia.log](https://github.com/xenia-project/game-compatibility/files/2043018/xenia.log)

# Screenshot(s):
![image](https://user-images.githubusercontent.com/4259439/40590718-e713acfe-6203-11e8-97a6-2b8e56962d36.png)

# Labels:
(state crash or white screen)

**Comment by illusion98** [comment]
Menus https://github.com/xenia-project/xenia/commit/f7ea93148a9ec9ff0ad9edac8c01854bd6153a10
Log [D3D12](https://cdn.discordapp.com/attachments/459064743877935105/597731658946445312/xenia.log)
![ ](https://cdn.discordapp.com/attachments/570608724205502464/597731380629209088/unknown.png)
![ ](https://cdn.discordapp.com/attachments/570608724205502464/597730983642529792/unknown.png)
![ ](https://cdn.discordapp.com/attachments/570608724205502464/597731045890195467/unknown.png)

**Comment by LittleStinky123** [comment]
It freezes after opening the redump ISO of my game with https://github.com/xenia-canary/xenia-canary/commit/988881267be1d8902db7b843cf8a787552c79914.
[xenia.log](https://github.com/xenia-project/game-compatibility/files/3794601/xenia.log)

**Comment by Margen67** [comment]
@LittleStinky123 Do other redump ISOs cause Xenia to freeze?

**Comment by illusion98** [comment]
The game is very unstable and getting in game is difficult.

**Comment by LittleStinky123** [comment]
@Margen67 No, all 3 of my Final Fantasy XIII-1 redump disc dumps work, Read Dead Redemption, MGS3 works and all were created using the same disc drive.

**Comment by Gliniak** [comment]
Tested on: https://github.com/xenia-project/xenia/commit/9f789e01b6ae60e306d3af44dcc1c0dc5d76d29c

# Overview:
Game requires `disable_global_lock = true`

- Game works quite nice, audio is fine.
- During cutscenes models likes to flicker.
- Sometimes game still can hang on loading.
- Saving doesn't work (or loading?)

# Log:
(maybe later)

# Screenshots:
![image](https://user-images.githubusercontent.com/153369/84811209-cdf98780-b00c-11ea-91b1-5eea2e5a206b.png)
![image](https://user-images.githubusercontent.com/153369/84812392-a4416000-b00e-11ea-8482-354d0a13f893.png)

**Comment by Arka161** [comment]
The game straight up crashes a few seconds after going into gameplay. I have an i7 107050H and an RTX 2060.

**Comment by Iammethe24** [comment]
Seems like the game can play flawlessly till the first point you can save without any changes to config. I don't see characters flickering either. Further testing is needed. (93b5886)

**Comment by Iammethe24** [comment]
> The game straight up crashes a few seconds after going into gameplay. I have an i7 107050H and an RTX 2060 (the opening cutscene works good).

Try switching to canary.
